recipes/iso8601lib/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conan import ConanFile
+from conan.errors import ConanException
+from conan.tools.files import get, copy, collect_libs
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class Iso8601LibConan(ConanFile):
+    name = "iso8601lib"
+    description = "Parsing a subset of ISO 8601 datetimes, dates and times in C."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/TimSC/iso8601lib"
+    topics = ("iso8601", "date", "time", "timezone")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        # To install relocatable shared libs on Macos
+        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0042"] = "NEW"
+        # Honor BUILD_SHARED_LIBS from conan_toolchain (see https://github.com/conan-io/conan/issues/11840)
+        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.libs = collect_libs(self)
+
+        self.cpp_info.filenames["cmake_find_package"] = self.name
+        self.cpp_info.filenames["cmake_find_package_multi"] = self.name
+        self.cpp_info.names["cmake_find_package"] = self.name
+        self.cpp_info.names["cmake_find_package_multi"] = self.name
+
+        self.cpp_info.defines.append("ISO8601LIB_EXPORT")
```

Comment:
This does not install the exported CMake targets https://github.com/TimSC/iso8601lib/blob/master/CMakeLists.txt to a config file so these are not needed

```suggestion
        self.cpp_info.defines.append("ISO8601LIB_EXPORT")
```